### ISSUE TITLE
Index multiple blocks in parallel

### ISSUE THREAD

**Issue by makeholer** [iquidus/explorer#247]
I've tested all explorer's over there, Iquidus seems to be the faster one for query, but it's impossible to save all data from bitcoin network because it's slow on processing and saving the transactions, other explorers like MouseExplore can analyze multiple blocks at same time running the process multiple times, i really suggest doing something like this on Iquidus, so it will be faster and will be usable for Bitcoin network...

**Comment by OttoKoester** [comment]
I've a real problem to index an old blockchain.
Could someone explain if it's possible to index multiple blocks, like oldest in one task and the latest blocks in other task. Thanks.

**Comment by uaktags** [comment]
It's possible to setup a cluster on sync.js. I can provide the code that I've used to test with, but as I mention in #252 I've yet to find any real success with performance improvements (albeit I only did a single coin, and my method of assigning blocks wasn't ideal). I could probably do what @OttoKoester is asking by making one worker start from 1+ and another worker start from stats.count (last) and go backwards. The thing I'm still trying to figure out with nodejs cluster is how to share information between the workers or atleast track progress.

Perhaps if we reduce ./bin/cluster from using .cpus().length and instead half it, we can then have more workers with full power to run sync. Then starting from the beginning wouldn't be as bad as a single process.
